hack/make-tools.mk

```diff
@@ -1,13 +1,8 @@
 # go-get-tool will 'go get' any package $2 and install it to $1.
 define go-get-tool
 @[ -f $(1) ] || { \
-set -e ;\
-TMP_DIR=$$(mktemp -d) ;\
-cd $$TMP_DIR ;\
-go mod init tmp ;\
```

Comment:
I never quite understood why the scripts were generated this way. If you can provide some description as to what they were trying to do and why we don't need it, that would make me feel better about this.

Reply:
added it in the description.